frontend-react/src/components/Admin/EditSenderSettings.test.tsx

```diff
@@ -64,8 +65,15 @@ const testKeys = JSON.stringify([
 const testProcessingType = "sync";
 
 describe("EditSenderSettings", () => {
-    beforeAll(() => settingsServer.listen());
-    afterEach(() => settingsServer.resetHandlers());
+    beforeAll(() => {
+        settingsServer.listen();
+        settingsServer.use(
+            rest.get(
+                "https://test.prime.cdc.gov/api/settings/organizations/abbott/senders/user1234",
```

Comment:
I'm wondering if we should use the `REACT_APP_BACKEND_URL` environment variable for the base path here.  Also, I think it'd be good to make sure this is an invalid base path so we're not actually making calls to test.prime.cdc.gov since that's real.

Not an issue with Jest tests since they should be sandboxed, but I'm thinking we could adopt the pattern for all tests, including Cypress. 🤔

Reply:
Funny enough, I'm pretty sure with this msw update preflight calls were succeeding to test.prime.cdc.gov which only stopped once I started isolating hook tests.